src/sentry/api/endpoints/organization_release_files.py

```diff
@@ -100,7 +100,7 @@ def post(self, request, organization, version):
         fileobj = request.FILES['file']
 
         full_name = request.DATA.get('name', fileobj.name)
-        if not full_name:
+        if not full_name or full_name == 'file':
```

Comment:
in 1.8 the file has a default name that i think is based on the key, but either way is 'file'